### PR TITLE
Chore: Improve Ledger docs

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Remember how to get the Ledger device model in case we need it.

In this PR, I add a comment to the `getVersion` method. I thought that was the easiest way to keep track of that instead of some documentation somewhere else.

# Changes

* Extend comment in `LedgerIdentity#getVersion` method.

# Tests

No code changes.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4417** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4417?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4417?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/lib/identities/ledger.identity.ts [hunk 1]
```diff
@@ -145,7 +145,14 @@ export class LedgerIdentity extends SignIdentity {
   }
 
   /**
-   * @returns The version of the `Internet Computer' app installed on the Ledger device.
+   * @returns The version of the `Internet Computer' app installed on the Ledger device and the device model used under the `targeId` key:
+   *
+   * 3110 => LNS
```
Comment (dskloetd) [inline frontend/src/lib/identities/ledger.identity.ts:150]: These are not numbers we get. These are prefixes of hex numbers.
In Slack, Carlo wrote "Using the first 4 chars (two bytes) of it".
Can you replace `3110` with `0x3110????` etc?

And can you also link to https://github.com/LedgerHQ/blue-loader-python?tab=readme-ov-file#target-id

**Review by dskloetd — COMMENTED** [review]
Thanks!

File: frontend/src/lib/identities/ledger.identity.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/identities/ledger.identity.ts]: Done!

**Comment by lmuntaner** [comment]
Done, ready for another review

**Review by dskloetd — APPROVED** [review]
Thanks!

**Pull request merged**